spec/support/utilities_spec.rb

```diff
@@ -1,3 +1,16 @@
-require 'support/sign_in_helper_spec'
```

Comment:
I'm not sure what these support files do but apparently something goes wrong here and the "sign in" examples in navigation_spec.rb fail. What I found out is, that support/sign_in_helper_spec.rb and support/utilities_spec.rb have the same content. Probably that is what is responsable for the failing tests. Probably you want to check that and discuss it with your coach.